libs/core/schematics/utils/file-utils.ts

```diff
@@ -1,10 +1,11 @@
 import { join, Path } from '@angular-devkit/core';
 import { Tree } from '@angular-devkit/schematics';
 
-const STYLESHEET_REGEX = /.*\.(css|scss|less)/;
+const STYLESHEET_REGEX = /.*\.(css|scss)/;
```

Comment:
What do we have against "sass" and "less" extensions?

Reply:
nothing :D 

fixed